core/src/main/java/io/questdb/cutlass/pgwire/modern/PGPipelineEntry.java

```diff
@@ -1194,7 +1181,7 @@ private void copyParameterValuesToBindVariableService(
                         setBindVariableAsDate(i, lo, valueSize, bindVariableService, characterStore);
                         break;
                     case X_PG_BOOL:
-                        setBindVariableAsBoolean(i, valueSize, bindVariableService);
+                        setBindVariableAsBoolean(i, lo, valueSize, bindVariableService);
```

Comment:
we must check that valueSize is withing the msgLimit

Reply:
done